app/views/reviews/_review.html.haml

```diff
@@ -0,0 +1,9 @@
+%div.review-container
```

Comment:
`%div.review-container` can be written as `.review-container` since `%div` is implicit